runtime/cmake/platform/os/zos.cmake

```diff
@@ -25,7 +25,10 @@ set(CMAKE_SHARED_LIBRARY_PREFIX "lib")
 
 set(CMAKE_IMPORT_LIBRARY_PREFIX "lib")
 
-list(APPEND OMR_PLATFORM_COMPILE_OPTIONS
-    "\"-Wc,inline(auto,noreport,600,5000)\""
-    "\"-Wc,list(),offset,gonumber\""
-)
+# todo: perhaps this belongs in corresponding toolchain file?
+if($(TOOLCHAIN_TYPE),xlc)
```

Comment:
Variables are referenced using `${var}`, the toolchain is in `OMR_TOOLCONFIG` and this comparison needs to use `STREQUAL`:
```
if(${OMR_TOOLCONFIG} STREQUAL "xlc")
```